README.md

```diff
@@ -1324,6 +1324,16 @@ workspace:
     # which breaks down the statistics by warning code.
     statVerbosity: "compact-stats"
 
+  # Optional globs to define paths to packages in workspace.
+  # Will default to [ "**/spago.yaml" ]
+  includedPackages:
+    - "apps/server/*/spago.yaml"
+    - "apps/client/*/spago.yaml"
+
+  # Option to search for packages in gitignored files
+  # Will default to false
+  searchForPackagesInIgnoredFiles: true
```

Comment:
My 2 cents :coin::coin:
this flag shouldn't be necessary, I would expect `includedPackages` to override `.gitignore` patterns that would otherwise ignore those globs

Reply:
Yeah that seems reasonable. I don't have a strong view on this so happy to go either way.